packages/rlc-common/src/helpers/importsUtil.ts

```diff
@@ -55,7 +55,7 @@ export function buildRuntimeImports(flavor?: PackageFlavor): Imports {
       commonFallback: {
         type: "commonFallback",
         specifier: "@typespec/ts-http-runtime",
-        version: "1.0.0-alpha.20240314.2"
+        version: "^0.1.0"
```

Comment:
let's fix for v0.1.0 for now.

```suggestion
        version: "0.1.0"
```